geograypher/cameras/cameras.py

```diff
@@ -690,46 +705,69 @@ def get_lon_lat_coords(self):
         """Returns a list of GPS coords for each camera"""
         return [x.get_lon_lat() for x in self.cameras]
 
+    def get_camera_locations(self):
+        return [x.get_camera_location() for x in self.cameras]
+
     def get_subset_ROI(
         self,
-        ROI: Union[PATH_TYPE, gpd.GeoDataFrame],
-        buffer_radius_meters: float = 50,
-    ):
+        ROI: Union[PATH_TYPE, gpd.GeoDataFrame, Polygon, Multipolygon],
+        buffer_radius: float = 0,
+        is_geospatial: bool = None
+    ): #TODO: wrap docstring at 100 
         """Return cameras that are within a radius of the provided geometry
 
         Args:
-            geodata (Union[PATH_TYPE, gpd.GeoDataFrame]): Geopandas dataframe or path to a geofile readable by geopandas
-            buffer_radius_meters (float, optional): Return points within this buffer of the geometry. Defaults to 50.
+            geodata (Union[PATH_TYPE, gpd.GeoDataFrame, Polygon, Multipolygon]): 
+                Geopandas dataframe, path to a geofile readable by geopandas, or Shapely Polygon
+                /Multipolygon information that can be loaded into a geodataframe
+            buffer_radius (float, optional): 
+                Return points within this buffer of the geometry. Defaults to 0. Represents 
+                meters if ROI is geospatial.
+            is_geospatial (bool, optional): flag for user to indicate if ROI is geospatial or not
         """
-        if not isinstance(ROI, gpd.GeoDataFrame):
+        # construct GeoDataFrame if not provided
+        if isinstance(ROI, Polygon) or isinstance(ROI, Multipolygon): 
+            if is_geospatial:
+                ROI = gpd.GeoDataFrame(index=[0], crs='epsg:4326', geometry=[ROI])
```

Comment:
This probably deserves a comment explaining that you're assuming it's lat-lon. I personally might error in this situation instead, but this is reasonable.